packages/app/galaxy/datatypes/converters

```diff
@@ -0,0 +1 @@
+../../../../lib/galaxy/datatypes/converters
```

Comment:
I think this belongs in galaxy-data also unless you want to refactor to a new package structure.